widget/calendar.go

```diff
@@ -43,30 +46,52 @@ type Calendar struct {
 // Since: 2.6
 func NewCalendar(cT time.Time, changed func(time.Time)) *Calendar {
 	c := &Calendar{
-		currentTime: cT,
-		OnChanged:   changed,
+		displayedMonth: cT,
+		OnChanged:      changed,
 	}
 
 	c.ExtendBaseWidget(c)
 	return c
 }
 
+// SetDisplayedMonth sets the currently displayed year and month
```

Comment:
Let's add a terminating dot at the end of the sentence and a `Since: 2.7` line two rows below.